### ISSUE TITLE
Disabling administrative shares breaks PsTools

### ISSUE THREAD

**Issue by zdat** [undergroundwires/privacy.sexy#249]
I'm trying to use something called [Pstools]([url](https://windowsreport.com/user-account-permission-disable-task/)) to disable scheduled tasks that you normally can't disable through the regular Task Scheduler. It opens up a special elevated Task Scheduler that can be used to disable stubborn tasks.

Something in the Privacy.sexy script is causing the error
```
Error creating key file on DESKTOP-XXXXXXX:
The network name cannot be found.
```

to display in the cmd window when I try to get the elevated Task scheduler to open. Does anyone know what it could be? Thanks.

**Comment by zdat** [comment]
Unchecking / reverting "disable administrative shares" fixes it.

**Comment by undergroundwires** [comment]
Hi @zdat, thank you for the report and sharing the root cause. I guess we should move recommendation level from `Standard` to `Strict`. We do not want `Standard` to break stuff.

**Comment by SnowzNZ** [comment]
Add [this ](https://learn.microsoft.com/en-us/troubleshoot/windows-server/networking/remove-administrative-shares)as a doc
Also this doc specifies a different registry key (AutoShareServer) instead of AutoShareWks.

**Comment by undergroundwires** [comment]
Script is documented and removed from Standard pool.

![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/80edd164-00b8-475e-86f9-af6306ac5555)

Released in [0.12.9](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.12.9) 🚀

**Comment by undergroundwires** [comment]
Script is documented and removed from Standard pool.

![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/80edd164-00b8-475e-86f9-af6306ac5555)

Let me know if you see another side-effect or just any other input.

Released in [0.12.9](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.12.9) 🚀
